tools/packaging/static-build/kernel/Dockerfile

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
Out of curiosity, why do we need to bump to 22.04?

Reply:
I am building a 22.04 rootfs, kernel and modules need to be built with the same kernel. Only 22.04 has the open-source drivers shipped.

Reply:
We should anticipate switching all the builds to 22.04 aka jammy.

Reply:
Yeah, I have no issues about that, that was just to understand the why. :-)